app/qml/main.qml

```diff
@@ -287,7 +327,12 @@ ApplicationWindow {
             mapCanvas.mapSettings.setCenter(positionKit.projectedPosition)
         }
 
-        onRemoveClicked: stateManager.state = "view"
+        onRemoveClicked: {
+            if (stateManager.state === "edit") {
+                featurePanel.show_panel(featurePanel.feature, "Edit", "form")
+            }
+            stateManager.state = "view"
+        }
     }
 
     RecordCrosshair {
```

Comment:
Right now the crosshair is below the GPS location marker - I think it should be on top of it.